config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :recipes
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root 'welcome#index'
```

Comment:
Were the corresponding views and controller removed?

Reply:
Not answered

Reply:
Yup, because of misunderstanding of solving conflicts in git. My fault boss

Reply:
We still have `app/controllers/welcome_controller.rb` and its corresponding views. Please remove.